### ISSUE TITLE
Form Config object ticket for content-build

### ISSUE THREAD

**Issue by derekhouck** [department-of-veterans-affairs/va.gov-team#89759]
## Overview
Once the Drupal data has been normalized, (see #88634) we need to format the output so it conforms to something `vets-website` can use. There should be a [Form Config object](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-form-config-options), but we also need additional data not contained in the Form Config, like the form's ID and OMB Number.

### Example
Here's an example of what that output might look like.
```json
[
  {
    "id": 12345,
    "ombNumber": 1234-5678,
    "formConfig": {
       "title": "My form",
       ...
       "chapters": {
         ...
       }
    }
  },
  ...
]
```

## Acceptance Criteria
- [ ] The output contains a valid Form Config object for each form
- [ ] The output contains additional fields for each form like "id" and "ombNumber"

**Comment by derekhouck** [comment]
Turns out, while schemas can be imported from the [vets-json-schema](https://github.com/department-of-veterans-affairs/vets-json-schema) repo, uiSchemas are all currently located within the [vets-website](https://github.com/department-of-veterans-affairs/vets-website) repo and are not easily exported. This makes it very difficult to export full Form Config objects from `content-build` without essentially duplicating the code for all the uiSchemas currently housed in vets-website.

**Comment by derekhouck** [comment]
I spoke with @ryguyk about this and proposed three paths forward:

1. We create a partial `formConfig` in `content-build` and send it over along with enough context that `vets-website` can create the rest. This offloads some of the business logic to the backend, but divides the `formConfig` creation amongst two repos. This makes it very difficult to use the output of `content-build` for anything other than this specific `vets-website` application.
2. We output a normalized data structure from `content-build` (essentially the output of #88634) and `vets-website` handles all aspects of `formConfig` creation. This creates cleaner business logic but places more of a burden on the frontend. While `content-build` does not output a Form Config object, it would output an easily-readable structure that other applications could theoretically build off of.
3. We (or the VFF team) transfer all existing uiSchemas from `vets-website` into `vets-json-schema`. This would allow `content-build` (or any other repo) to easily pull in the uiSchemas and create full Form Config objects without the need to integrate with `vets-website`. This is not a small project though, and runs into jurisdictional issues with the Forms team.

**Comment by derekhouck** [comment]
Ultimately, Ryan and I were in agreement that option 2 was the best path forward for us at this time. As a consequence, I will eventually be closing this ticket, as it no longer reflects the work we will be doing. I will be creating a new ticket to document the work of creating the Form Config object within the `vets-website` repo. I will keep this ticket and the branch I created for it open long enough to transfer over any applicable code/business logic that I can from the work already done, and I'll be sure to link the new ticket to this one as soon as it is created.

I will also be reaching out to the VFF team through their Slack channel. While we are not pursuing option 3 at this time, we still think there is value in letting the team know about this roadblock we encountered, and we agree that pulling the uiSchemas out of `vets-website` is a worthwhile long-term goal that should end up on the Forms team's backlog.
